WIP: Starknet Snap Rough Skeleton (#1509)
MetaMask/metamask-docs: wallet/how-to/use-non-evm-networks/starknet/tutorial.md

---
description: Starknet integration tutorial
sidebar_position: 6
---

> # Build a Basic dApp with get-starknet and React TypeScript

Reviewer: ```suggestion
# Create a simple Starknet dapp
```

Simplify title for cleaner sidebar.

> In this tutorial, you'll learn how to set up a basic dApp that uses get-starknet to connect to MetaMask and display the user's wallet address.

Reviewer: ```suggestion
In this tutorial, you'll learn how to set up a basic ddpp that uses the `get-starknet` library to connect to MetaMask and display the user's wallet address.
```

> 

Reviewer: Add prerequisites? yarn, text editor, etc.

> ## 1. Project Setup

Reviewer: ```suggestion
## 1. Set up the project
```

It looks like this tutorial might be copy-paste and needs an editing pass? I'll refrain from adding more comments until it's more ready. Headings should be sentence case and avoid gerunds. Admonitions should be properly formatted, etc.
